webknossos-jni/src/dracoToStlDecoder.cpp

```diff
@@ -39,10 +51,25 @@ JNIEXPORT jbyteArray JNICALL Java_com_scalableminds_webknossos_datastore_draco_N
                 positionAttribute->GetMappedValue(face[0], &pos[0][0]);
                 positionAttribute->GetMappedValue(face[1], &pos[1][0]);
                 positionAttribute->GetMappedValue(face[2], &pos[2][0]);
+
+                if(vertexQuantizationBits > 0) {
+                  auto vertexQuantizationFactor = 1 / (pow(2, double(vertexQuantizationBits)) - 1);
+                  pos[0][0] = float(extractUint16(pos[0][0])) * vertexQuantizationFactor;
+                  pos[0][1] = float(extractUint16(pos[0][1])) * vertexQuantizationFactor;
+                  pos[0][2] = float(extractUint16(pos[0][2])) * vertexQuantizationFactor;
+                  pos[1][0] = float(extractUint16(pos[1][0])) * vertexQuantizationFactor;
+                  pos[1][1] = float(extractUint16(pos[1][1])) * vertexQuantizationFactor;
+                  pos[1][2] = float(extractUint16(pos[1][2])) * vertexQuantizationFactor;
+                  pos[2][0] = float(extractUint16(pos[2][0])) * vertexQuantizationFactor;
+                  pos[2][1] = float(extractUint16(pos[2][1])) * vertexQuantizationFactor;
+                  pos[2][2] = float(extractUint16(pos[2][2])) * vertexQuantizationFactor;
+                }
+
                 draco::Vector3f norm = draco::CrossProduct(pos[1] - pos[0], pos[2] - pos[0]);
                 norm.Normalize();
                 encodeBuffer.Encode(norm.data(), sizeof(float) * 3);
 
+
```

Comment:
Could you run clang-format on this file please? There is already a `.clang-format` file in `webknossos-jni/src` with some rules for the formatter.